optuna/visualization/_rank.py

```diff
@@ -1,12 +1,10 @@
+from __future__ import annotations
+
 import math
 import typing
 from typing import Any
 from typing import Callable
-from typing import List
```

Comment:
We need to update `_TickInfo` 's members to remove `List`.

```
class _TickInfo(NamedTuple):
    coloridxs: List[float]
    text: List[str]
```

Reply:
Thanks for your comment. I updated them.

Reply:
Thanks!